src/transformers/hyperparameter_search.py

```diff
@@ -13,9 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from packaging.version import Version
```

Comment:
To be consistent with other occurrences in transformers

```suggestion
from packaging import version
```